MAUI/DataGrid/Column-types.md

```diff
@@ -408,6 +408,65 @@ Underlying records will be the BindingContext for the `CellTemplate`.
 {% endhighlight %}
 {% endtabs %}
 
+### Load edit template to the template column
+
+The `SfDataGrid` allows you to load any custom view in edit mode using the [EditTemplate]() property.
+
+{% highlight xaml %}
+    <ContentPage.Content>
```

Comment:
Use **Order ID**, **Customer ID**, **Ship Country**, and **Is Confirmed** columns. **Is Confirmed** column should be a template column and display the true and false values. In edit mode, the checkbox should be loaded.

Reply:
Had Updated the Code Snippet